arangodb-net-standard/TransactionApi/TransactionApiClient.cs

```diff
@@ -48,11 +48,21 @@ public TransactionApiClient(IApiClientTransport client, IApiClientSerialization
         /// </summary>
         /// <typeparam name="T">Type to use for deserializing the object returned by the transaction function.</typeparam>
         /// <param name="body">Object containing information to submit in the POST transaction request.</param>
+        /// <param name="serializationOptions">Optional serialization options.</param>
         /// <returns>Response from ArangoDB after processing the request.</returns>
         public virtual async Task<PostTransactionResponse<T>> PostTransactionAsync<T>(
-            PostTransactionBody body)
+            PostTransactionBody body,
+            ApiClientSerializationOptions serializationOptions = null)
         {
-            var content = GetContent(body, new ApiClientSerializationOptions(true, true));
+            // Set default serialization options if not passed in
+            serializationOptions = serializationOptions ??
+                new ApiClientSerializationOptions(
+                     useCamelCasePropertyNames: true,
+                     ignoreNullValues: true,
+                     useStringEnumConversion: true,
```

Comment:
Should we be consistent with how `DefaultOptions` are defined in `ApiClientSerialization`? For all other endpoints we have `useStringEnumConversion` set to `false` by default.

Default options:
```
new ApiClientSerializationOptions(false, true, false);
```

Reply:
I think you might be right, I'll update